website.py

```diff
@@ -1,4 +1,4 @@
 import sys
-import os.path
+import os
```

Comment:
If you're using nothing else from the `os` module, import `os.path`. [It's the recommended approach](https://stackoverflow.com/questions/2724348/should-i-use-import-os-path-or-import-os).